tests/LineScanCameraTests.cpp

```diff
@@ -34,12 +34,15 @@ TEST_F(ConstVelocityLineScanSensorModel, Center) {
 }
 
 TEST_F(ConstVelocityLineScanSensorModel, Inversion) {
+  double *achievedPrecision = new double;
```

Comment:
You can just pass a pointer to a local variable here, no need to deal with heap allocation.